frontend/src/components/Month/Dates.tsx

```diff
@@ -0,0 +1,38 @@
+import moment, { now } from "moment";
+
+type Dtype = {
+  month: number;
+  date: number;
+  day: number;
+};
+
+export const getDates = (criteria: moment.Moment) => {
+  const startDate = criteria.clone();
+  const dates: Dtype[][] = new Array(6);
+
+  for (let i = 0; i < dates.length; i++) dates[i] = new Array();
```

Comment:
위의 두 코드를 아래처럼 하나의 코드로 작성할 수 있습니다!
```suggestion
  const dates: Dtype[][] = Array.from(Array(6), () => []);
```